docs/release-notes/snapcraft-8-13.rst

```diff
@@ -46,3 +59,116 @@ the following commands in your local repository to sync the submodule change:
     git submodule sync
     git submodule update --init --recursive
     git clean -ffd
+
+
+New .NET extensions
+~~~~~~~~~~~~~~~~~~~
+
+The experimental ``dotnet8``, ``dotnet9``, and ``dotnet10``
+:ref:`reference-dotnet-extensions` have been added to Snapcraft for ``core24`` snaps.
+
+
+Minor features
+--------------
+
+Old command aliases
```

Comment:
Is this really a minor change? This affects a swath of commands that users invoke frequently.

Reply:
But we want users to switch, yes? As in, going forward and canonically, all of our material will promote commands without `list`?